devel/fetch_metrics/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
```

Comment:
Here's the docker recipe for spiketoolkit. Note the version label at the bottom. In order to use with singularity, the image must be on dockerhub. The label version must coincide with the tag on dockerhub